src/components/counter.js

```diff
@@ -0,0 +1,90 @@
+// @ts-check
+
+import React, { Component } from 'react'; 
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import {StyleSheet, Text, View} from 'react-native'
+import { Button } from 'react-native-elements'
+
+import {counterIncrement, counterDecrement}  from '../actions/counter';
+
+const styles = StyleSheet.create({ 
+    container: { 
+        flex: 1, 
+        padding: 30, 
+        justifyContent: 'center', 
+        alignItems: 'center', 
+        backgroundColor: '#000080',
+    },
+
```

Comment:
Remove any inconsistent spacing. I suggest using a tool like https://github.com/prettier/prettier. It has a VS code plugin here: https://github.com/prettier/prettier-vscode. It will auto format your code for you, as it's super tedious to do it manually.